tests/test.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
This module does not contain any actual unit test, so I suggest it is named `tests/helpers.py`. It should be imported in all test modules as `from .helpers import ...`.